lib/pennyworth/cli.rb

```diff
@@ -141,13 +141,13 @@ def self.settings= s
     LONGDESC
     arg_name "IMAGE_NAME", :optional
     command "build-base" do |c|
-      c.flag [:kiwi_tmp_dir, :k], :type => String, :required => false,
-        :desc => "Temporary KIWI directory for building the Vagrant box.",
-        :arg_name => "KIWI-TMP-Dir"
+      c.flag [:temp_dir, :t], :type => String, :required => false,
+        :desc => "Temporary directory for building the Vagrant box.",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.